app/pages/system/ip-pools/IpPoolPage.tsx

```diff
@@ -0,0 +1,176 @@
+      <Message
+        className="-mt-8 mb-12"
+        variant="info"
+        content="This page is a work in progress. Use the CLI or API for full control over IP ranges and linked silos."
+      />
+      <QueryParamTabs className="full-width" defaultValue="ranges">
+        <Tabs.List>
+          <Tabs.Trigger value="ranges">IP ranges</Tabs.Trigger>
+          <Tabs.Trigger value="silos">Linked silos</Tabs.Trigger>
+        </Tabs.List>
+        <Tabs.Content value="ranges">
+          <IpRangesTable />
+        </Tabs.Content>
+        <Tabs.Content value="silos">
+          <LinkedSilosTable />
+        </Tabs.Content>
+      </QueryParamTabs>
+    </>
+  )
+}
+
+const RangesEmptyState = () => (
+  <EmptyMessage
+    icon={<Networking24Icon />}
+    title="No IP ranges"
+    body="Add a range to see it here"
+    // TODO: link add range button
+    // buttonText="Add range"
+    // buttonTo={pb.ipPoolNew()}
+  />
+)
+
+function IpRangesTable() {
+  const poolSelector = useIpPoolSelector()
+  const { Table, Column } = useQueryTable('ipPoolRangeList', { path: poolSelector })
+
+  return (
+    <Table emptyState={<RangesEmptyState />}>
+      {/* TODO: only showing the ID is ridiculous. we need names */}
+      <Column accessor="range.first" header="First" />
+      <Column accessor="range.last" header="Last" />
+      <Column accessor="timeCreated" header="Created" cell={DateCell} />
+    </Table>
+  )
+}
+
+const SilosEmptyState = () => (
+  <EmptyMessage
+    icon={<Networking24Icon />}
+    title="No IP pool associations"
+    body="You need to link the IP pool to a silo to be able to see it here"
+    // TODO: link silo button
+    // buttonText="Link IP pool"
+    // buttonTo={pb.ipPoolNew()}
+  />
+)
+
+function SiloNameFromId({ value: siloId }: { value: string }) {
+  const { data: silo } = useApiQuery('siloView', { path: { silo: siloId } })
+
+  if (!silo) return <SkeletonCell />
+
+  return <LinkCell to={pb.siloIpPools({ silo: silo.name })}>{silo.name}</LinkCell>
+}
+
+function LinkedSilosTable() {
+  const poolSelector = useIpPoolSelector()
+  const queryClient = useApiQueryClient()
+  const { Table, Column } = useQueryTable('ipPoolSiloList', { path: poolSelector })
+
+  const unlinkSilo = useApiMutation('ipPoolSiloUnlink', {
+    onSuccess() {
+      queryClient.invalidateQueries('ipPoolSiloList')
+    },
+  })
+
+  // TODO: confirm action. make clear what linking means
+  const makeActions = (link: IpPoolSiloLink): MenuAction[] => [
+    {
+      label: 'Unlink',
+      onActivate() {
+        unlinkSilo.mutate({ path: { silo: link.siloId, pool: link.ipPoolId } })
+      },
+    },
+  ]
+
+  return (
+    <>
+      <p className="mb-8 max-w-2xl text-sans-md text-secondary">
+        Users in linked silos can allocate external IPs from this pool for their instances.
+        A silo can have at most one default pool. IPs are allocated from the default pool
+        when users ask for one without specifying a pool.
+      </p>
```

Comment:
@benjaminleonard @paryhin check out this big ugly paragraph

https://console-git-ip-pools-ui-oxidecomputer.vercel.app/system/ip-pools/ip-pool-1?tab=silos

Reply:
Yeah, I'm sure you can remove. Perhaps we add a note in the release notes that this section will be changing shortly?